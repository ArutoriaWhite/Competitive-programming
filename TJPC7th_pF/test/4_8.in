11872805 991
23770 0
24049 0
40767 0
58263 1
65536 1
72427 0
90526 1
137349 1
145917 1
181614 1
186432 1
198073 1
198423 1
205207 1
218430 1
220937 0
244007 0
248676 0
293864 1
294057 1
296259 1
301562 1
301841 1
302746 0
309297 1
317704 1
348947 1
386245 0
387521 0
397326 0
398916 0
405433 1
408587 0
441707 0
449765 1
457447 1
475896 0
476837 0
488391 1
493738 0
503371 0
504282 1
510843 1
519727 0
520533 0
534489 1
544663 1
553220 1
561753 1
567957 1
594261 0
605856 1
635545 1
636589 1
638655 1
641276 0
643176 1
652362 0
654570 0
671929 0
698177 0
701220 0
711848 0
722021 1
722827 0
732887 1
743234 1
761256 1
805143 0
840307 1
840478 1
857592 0
870487 1
883750 0
883758 0
891935 0
903678 1
908380 1
916365 1
936944 1
951888 0
954771 0
959910 0
962586 1
974459 0
977824 1
990445 0
1002650 1
1012693 0
1022422 1
1040211 0
1048649 0
1054993 0
1056135 0
1065512 1
1100648 1
1139660 1
1146764 1
1151093 1
1165951 1
1183148 0
1189916 1
1200358 0
1202496 0
1214610 1
1215587 1
1227098 1
1232346 1
1246625 0
1249843 1
1267850 1
1287984 0
1294298 0
1298978 1
1302576 1
1306350 0
1314190 1
1349857 0
1359285 1
1376603 1
1390524 1
1396494 1
1409821 1
1410493 1
1443952 1
1483162 1
1484940 1
1499870 1
1510120 1
1519415 0
1524935 1
1526729 0
1541257 1
1579352 0
1584160 0
1590728 0
1604969 1
1605498 1
1617682 1
1641668 0
1679510 0
1705631 0
1711116 1
1713443 1
1715524 0
1716825 1
1727737 0
1728983 1
1731910 1
1751530 0
1753070 0
1764435 0
1766363 0
1766993 1
1770414 0
1790978 1
1795392 0
1800106 1
1807796 1
1810125 0
1814419 1
1822541 1
1822879 1
1834133 0
1835118 0
1837417 0
1849330 1
1855587 1
1894012 0
1905710 1
1922560 0
1935960 0
1944263 1
1970340 0
1974639 0
1978159 1
1986251 0
1997082 0
2003846 0
2018796 1
2035305 0
2040865 0
2051471 1
2054925 0
2057311 0
2085382 0
2112384 0
2121410 0
2133573 1
2136249 0
2138932 1
2194702 0
2195815 0
2202076 0
2216297 1
2216827 0
2217407 1
2224504 1
2227249 0
2229706 1
2230441 1
2238315 1
2238561 0
2260812 1
2267628 0
2281725 1
2288475 0
2319233 0
2321767 0
2328009 0
2372540 0
2377141 0
2415372 1
2421042 1
2424086 1
2468063 1
2480779 1
2481673 0
2488116 1
2492150 0
2498883 1
2514982 0
2530841 1
2539662 0
2547624 1
2553673 0
2569919 0
2580600 1
2581315 1
2591082 1
2599407 0
2605531 0
2625440 1
2649762 1
2669258 0
2672499 0
2691269 1
2691715 1
2716847 1
2718628 0
2723800 1
2765755 0
2795572 1
2801247 1
2803054 0
2804297 0
2809553 1
2809827 1
2810161 0
2810718 0
2813518 0
2845295 1
2868751 1
2937451 0
2944636 0
2953598 1
2960539 1
2961079 1
2976527 1
2989566 1
2993133 0
3008422 1
3011235 0
3013893 1
3021872 1
3023399 0
3042987 1
3048488 1
3049450 1
3061716 0
3062459 0
3066302 1
3068088 1
3083741 1
3089370 0
3091447 0
3115108 1
3120280 1
3124698 1
3158793 0
3161592 1
3180830 0
3226733 0
3252940 1
3265132 1
3271909 1
3275765 1
3277375 0
3285801 1
3291359 1
3291676 1
3294977 1
3319251 1
3330447 1
3335813 0
3348458 0
3356661 1
3368368 1
3400979 1
3445674 1
3447266 1
3448732 1
3455657 1
3479740 0
3488268 0
3492235 0
3495357 0
3497971 0
3508504 0
3512215 1
3535978 0
3539688 1
3551593 1
3590258 0
3633643 1
3653644 0
3668252 0
3677535 0
3700262 1
3723971 0
3728256 1
3737084 0
3758240 0
3762845 1
3772219 0
3776902 0
3781505 1
3796401 1
3807656 1
3809242 1
3829518 0
3845086 0
3848032 1
3848369 0
3850349 0
3851302 0
3853264 0
3854610 1
3910097 1
3931382 0
3948999 0
3965832 1
3966429 1
3967027 0
3968245 1
3974156 0
3975786 1
4014274 1
4022493 0
4026290 0
4040774 0
4041859 1
4043978 1
4051145 1
4053325 0
4087357 0
4089322 0
4097095 1
4112112 1
4132938 0
4149081 1
4183756 0
4196863 0
4198883 1
4202156 1
4218379 1
4225523 0
4234008 0
4240461 0
4257590 1
4268382 0
4334384 0
4350922 0
4366888 1
4369342 1
4387910 0
4399931 1
4405259 0
4439379 0
4454076 0
4501234 0
4516136 0
4520541 1
4526462 0
4529389 1
4559530 1
4570975 1
4575433 0
4580470 0
4607705 0
4609584 0
4609945 1
4620774 0
4625711 0
4627751 1
4632631 1
4644353 1
4653057 0
4654952 1
4656538 0
4675854 0
4686839 1
4694556 1
4699014 1
4715508 1
4719287 0
4720935 0
4721122 0
4743710 0
4744664 0
4759187 0
4770187 0
4833918 0
4835476 1
4892495 0
4905918 1
4911574 1
4928226 0
4936728 1
4939762 1
4942946 0
4944378 0
4980700 0
4983864 1
4987249 0
5002416 0
5011334 0
5022267 1
5026548 1
5030264 0
5032533 0
5064143 1
5085721 0
5086818 1
5087116 1
5087367 0
5093665 0
5110936 1
5111161 0
5116182 1
5121016 0
5125039 1
5137655 1
5159398 0
5209576 0
5248144 1
5254868 1
5267114 0
5297432 1
5323640 0
5331628 0
5348923 0
5351941 0
5362320 1
5363041 1
5378029 1
5378765 0
5396085 0
5409004 0
5423621 1
5425064 1
5439056 1
5441803 1
5444973 0
5452443 0
5463013 1
5470431 1
5494438 1
5513743 0
5527881 0
5537072 0
5537524 1
5537742 1
5549023 1
5579253 1
5583630 1
5621995 0
5626826 1
5640629 0
5643080 1
5659099 1
5671811 0
5673447 1
5682000 0
5709491 1
5710348 1
5720062 1
5728625 0
5728954 0
5730039 0
5730148 1
5733469 1
5763728 0
5768852 1
5769369 0
5786808 0
5799189 1
5841714 0
5852032 1
5852357 1
5862569 1
5885339 0
5890285 0
5901156 0
5913325 1
5915234 0
5943339 1
5972115 0
5977659 1
5980298 0
5999657 1
6010875 0
6030163 1
6037682 0
6059693 1
6083635 1
6083857 0
6096954 0
6101182 1
6105333 0
6109179 1
6109239 0
6122536 1
6122935 0
6133507 1
6152697 0
6153320 0
6167793 1
6172654 0
6175780 0
6182177 0
6187117 1
6196188 1
6229891 1
6233235 1
6249305 0
6297087 0
6313590 0
6315349 0
6325816 1
6332394 1
6339801 0
6352073 1
6382598 1
6382992 1
6392985 0
6395104 1
6397385 0
6405603 1
6438421 1
6447294 1
6452882 1
6466778 0
6491000 1
6504632 0
6506289 0
6507518 0
6507573 0
6513183 0
6518660 1
6533804 1
6547258 0
6552530 1
6556358 1
6570068 1
6602473 1
6620258 1
6624729 1
6695661 1
6706343 1
6710001 0
6712026 1
6725344 0
6736976 1
6763003 0
6773219 0
6778247 1
6779641 1
6793024 0
6795037 1
6796678 1
6798497 0
6804068 0
6823822 0
6825788 1
6831336 0
6852565 1
6861541 1
6870474 1
6891851 0
6918675 0
6935721 0
6943604 0
6951585 0
6960763 0
6993309 1
7012200 1
7087581 1
7094051 1
7100391 1
7109054 1
7125433 0
7132093 0
7136255 0
7146275 0
7156811 0
7169894 1
7195059 1
7203307 1
7204732 0
7212072 0
7228281 1
7244315 0
7267240 0
7271061 0
7285670 1
7308045 1
7319146 1
7323052 1
7324981 0
7325082 1
7331455 0
7341783 0
7353329 0
7355073 0
7369698 0
7370111 1
7370435 1
7371387 1
7371616 0
7400708 0
7413148 1
7418416 0
7425637 1
7437700 0
7454006 1
7457006 1
7461291 0
7488046 0
7501062 1
7511331 0
7518505 0
7521995 1
7532198 1
7535011 1
7554417 0
7556321 1
7572490 1
7581757 0
7609569 0
7635851 1
7638582 1
7662628 1
7673685 1
7689359 0
7694930 1
7695919 0
7701263 1
7705180 0
7721077 1
7727607 1
7755477 0
7762756 0
7763207 1
7786972 0
7804017 1
7823243 1
7849742 0
7856071 0
7861505 1
7864209 1
7876056 1
7901274 0
7907093 0
7911080 0
7928573 0
7931003 1
7935396 0
7941185 1
7954797 1
7958213 0
7966547 1
7975714 1
7977409 1
7977816 1
7981122 0
7984792 1
8013090 1
8022909 1
8029701 0
8040684 0
8055109 1
8067282 1
8085463 0
8115126 1
8135289 1
8148851 0
8159199 0
8165973 0
8169957 0
8170408 0
8179214 1
8212924 1
8232352 0
8233051 0
8254252 0
8266476 1
8273178 1
8277511 0
8289092 0
8293222 0
8294579 1
8295041 0
8319303 0
8319823 1
8323427 1
8360843 1
8383012 1
8383713 1
8392512 1
8418629 1
8428267 1
8445024 0
8451517 0
8454321 1
8463291 0
8466776 1
8466812 0
8469951 1
8472216 0
8473553 1
8493274 0
8539462 1
8544017 0
8556457 1
8584924 1
8589545 0
8590730 1
8591773 0
8602818 1
8609172 0
8639392 1
8652618 0
8672908 1
8676901 1
8679779 0
8697599 1
8713402 1
8725322 0
8726258 0
8738576 1
8754032 1
8754924 1
8757876 1
8792995 0
8815061 1
8817903 1
8824753 0
8840707 0
8848023 0
8854236 0
8862267 1
8898957 1
8918568 0
8933348 0
8935582 1
8947197 1
8960016 1
8974551 0
9010993 1
9015846 0
9017933 1
9051532 0
9054063 0
9105134 0
9119608 1
9125889 1
9126769 0
9147172 0
9147426 1
9153334 1
9154986 0
9160612 1
9208955 0
9209874 0
9216025 1
9226481 1
9231378 1
9248746 0
9278206 1
9279709 1
9291213 0
9291629 0
9301457 1
9312313 1
9318906 1
9330424 1
9355184 0
9368818 1
9369226 1
9386616 0
9412900 1
9433548 0
9443846 1
9458882 1
9461489 1
9464614 0
9469734 1
9470939 0
9476900 0
9493119 1
9500223 0
9500926 1
9530084 0
9532038 1
9536601 0
9537339 0
9568728 1
9627689 1
9632298 1
9633060 0
9636189 0
9639760 0
9656570 0
9661123 0
9690833 1
9694755 0
9712397 0
9723390 1
9736053 0
9742876 0
9784659 0
9791494 1
9822131 0
9833929 0
9908783 1
9909109 0
9914506 0
9915465 1
9952685 1
9953008 1
9965073 0
9969142 1
10000356 0
10005125 0
10010278 1
10022582 1
10032195 1
10042796 1
10045596 0
10081223 1
10082958 0
10083245 1
10089633 0
10098914 1
10106575 0
10149692 1
10171157 0
10171224 1
10193032 0
10215479 0
10216332 0
10223500 1
10243469 0
10263414 0
10276510 1
10281353 0
10324514 0
10329331 0
10334880 1
10339665 0
10347147 0
10347161 0
10368728 1
10369175 0
10373237 1
10393069 1
10487187 0
10489037 1
10493503 0
10498938 0
10507961 1
10511087 0
10514082 1
10520167 0
10533617 0
10538585 0
10565588 1
10581240 0
10582672 1
10616705 1
10616786 1
10630318 1
10636181 1
10636939 1
10638954 0
10645615 1
10648722 0
10651783 1
10685350 0
10693473 1
10716957 1
10725401 1
10725737 0
10758867 0
10778642 0
10836533 1
10837215 0
10848912 0
10850243 0
10856080 0
10864136 0
10881774 0
10885890 1
10896808 1
10926026 0
10942118 0
10951507 1
10954789 0
10968155 1
10974681 0
10975444 0
10979817 0
10980559 1
11014616 0
11024400 1
11035597 0
11036300 0
11088439 1
11092198 0
11145247 0
11158839 0
11166114 1
11171227 0
11178731 1
11196684 0
11208647 0
11211392 1
11224750 1
11242122 0
11242566 1
11247057 0
11251039 1
11273202 0
11276987 0
11311278 1
11319436 1
11339312 1
11340467 1
11346743 0
11370325 0
11370424 0
11389892 1
11393015 1
11408929 0
11419410 0
11437058 0
11452719 1
11493622 1
11493686 0
11510084 1
11513818 1
11514361 1
11516294 0
11521414 1
11526288 0
11533644 1
11535287 0
11540024 1
11540738 0
11545256 0
11571878 0
11580065 0
11648165 0
11648450 0
11650053 0
11650698 0
11663988 0
11678017 1
11686642 1
11723717 1
11769665 1
11772110 1
11775553 0
11806230 1
11809449 0
11811207 0
11824858 0
11845832 1
11848750 1
11862064 0
