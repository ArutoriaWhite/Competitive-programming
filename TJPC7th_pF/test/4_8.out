6993309 11749970
